<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="prev_screen_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Vissza az előző elemzésekhez</string>
   </property>
  </widget>
  <widget class="QLabel" name="top_word_label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>1030</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="top_word_label_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>1010</y>
     <width>371</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="next_screen_button">
   <property name="geometry">
    <rect>
     <x>1750</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>További elemzések</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>530</y>
     <width>1921</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="top_word_vbox_2"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1921</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="top_word_vbox_1"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
